### PR TITLE
External content mediawiki pages

### PR DESCRIPTION
**Pull request by Hyper-Node**
# MaRDI Pull Request

**Changes**:
- External pages can be imported in MediaWiki by placing something like: {{#embed:https://raw.githubusercontent.com/MaRDI4NFDI/formulasearch/main/README.md}} in  a pages source. Markdown is parsed. 
- Cyclic cron which invokes a maintenance script which refreshes the pages hourly in wikibase container.  

**Instructions for PR review**:
- [x] Conceptual Review (Logic etc...) 
- [x] Code Review (Review your implementation) 
- [ ] Checkout (Test changes locally) 

**Checklist for this PR**: 
- [x] [Reviewers and Assignee specified.](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users)
- [ ] [All related issues are linked.](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

### CODE REVIEW

File: LocalSettings.php.template [hunk 1]
```diff
@@ -49,8 +49,7 @@ ${DOLLAR}wgRCMaxAge = 365 * 24 * 3600;
 wfLoadSkin( 'Vector' );
 
 ## Wikibase
-# Load Wikibase repo & client with the example / default settings.
-require_once "${DOLLAR}IP/extensions/Wikibase/repo/Wikibase.php";
+wfLoadExtension( 'WikibaseRepository', "$IP/extensions/Wikibase/extension-repo.json" );
```
Comment (eloiferrer) [inline LocalSettings.php.template:52]: Shouldn't this be also with ${DOLLAR}?

File: LocalSettings.php.template [hunk 2]
```diff
@@ -49,8 +49,7 @@ ${DOLLAR}wgRCMaxAge = 365 * 24 * 3600;
...
 require_once "${DOLLAR}IP/extensions/Wikibase/repo/ExampleSettings.php";
-require_once "${DOLLAR}IP/extensions/Wikibase/client/WikibaseClient.php";
-require_once "${DOLLAR}IP/extensions/Wikibase/client/ExampleSettings.php";
+wfLoadExtension( 'WikibaseClient', "$IP/extensions/Wikibase/extension-client.json" );
```
Comment (eloiferrer) [inline LocalSettings.php.template:54]: Maybe also with ${DOLLAR}?

**Review by eloiferrer — APPROVED** [review]
If I am not mistaken there are a couple of $ on LocalSettings.php.template that should be substituted by ${DOLLAR}. The rest looks good.

File: LocalSettings.php.template (hunk 2 above)
Comment (Hyper-Node) [reply LocalSettings.php.template:54]: standard initialization from [Wikibase](https://www.mediawiki.org/wiki/Wikibase/Installation) is without dollar and it is working like this, so this is sufficient atm.

Comment (dajuno) [reply LocalSettings.php.template:54]: If there is an env var called `$IP`, envsubst in `entrypoint.sh` will substitute it. It's probably not the case, but to protect `$IP`, I'd still use `${DOLLAR}IP`. See https://stackoverflow.com/questions/24963705/is-there-an-escape-character-for-envsubst/24964089#24964089

Cf. here
https://github.com/MaRDI4NFDI/docker-wikibase/blob/bba24aefa1f9d4638bf84da86f4ebef782eb7102/entrypoint.sh#L40-L41

**Pull request merged**